### DISCUSSION TITLE
Can't control my zhimi.airpurifier.mb4

### DISCUSSION THREAD

**Discussion by NutchanonK** [rytilahti/python-miio#938] [Q&A]
I can't control my Air Purifier 3C with command. Like in a picture.
![ภาพ](https://user-images.githubusercontent.com/78018032/107242860-03048f80-6a5f-11eb-9cb9-4e4718ab4a5e.png)

Only one command thats work is info
![ภาพ](https://user-images.githubusercontent.com/78018032/107243197-5840a100-6a5f-11eb-8944-cf8865e84cf9.png)

**Comment by rytilahti** [comment]
Are you running the git master? Support for mb4 was added less than day ago with #899 by @arturdobo so you will need a recent git checkout. The command is also `miiocli airpurifiermb4` instead of `airpurifiermiot`, I think :-)

**Reply by arturdobo** [reply]
@NutchanonK  3C supports only a subset of i.e. 3/3H features, so that's why you need to use a different name in `miiocli`
